### ISSUE TITLE
[SF][A11y][JAWS][ui5-table]: Incorrect row count for the table in VPC and while arrowing down in forms mode

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#5755]
### **Bug Description**
Switch to JAWS VPC mode (INSERT+Z twice). 
Focus table header, you will hear:
Column 1, Row 1
Header Row Product Supplier Dimensions Weight Price 

Focus first row, you will hear:
Column 1, Row 2
Product Notebook Basic Supplier Very Best Screens Dimensions 30 x 18 x 3cm Weight 4.2KG Price 956EUR. 1 of 2

Please fix this issue in 1.7.x version.

### **Expected Behavior**
As table header also as a row, when Focus first row, the aria-label should be .......xxx...... 2 of 3.
The row count should be the same without any mismatch, whether we access the cells in forms or VPC mode.

### **Steps to Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-table--default-story'
2. Open JAWS with VPC mode

### **Context**
- UI5 Web Components version: {1.6.0}
- OS/Platform: {Windows 10}
- Browser: {Chrome latest/JAWS 2022}
- Affected component: {ui5-table}

### **Log Output / Stack Trace / Screenshots**
<img width="1787" alt="image" src="https://user-images.githubusercontent.com/13098801/187453563-04c649de-1ef5-48b7-84e1-e877faccc6dd.png">

This video is from SF A11y testing team.
https://user-images.githubusercontent.com/13098801/187455822-2fd5ed94-a5a4-41d2-9f9f-25ee4434dc28.mp4

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SAP SuccessFactors}
- Business impact: {...}

**Comment by ndeshev** [comment]
Hello @SAP/ui5-webcomponents-topic-rl the announcement of the row count is different when the JAWS VPC mode is on and off, it can be reproduced in the example under the title "3 columns, but 1 cell" here:
https://sap.github.io/ui5-webcomponents/playground/main/pages/Table
